app-api/src/main/java/org/openmrs/eip/app/management/entity/SyncResponse.java

```diff
@@ -0,0 +1,80 @@
+package org.openmrs.eip.app.management.entity;
+
+import java.util.Date;
+
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "sync_response")
+public class SyncResponse extends AbstractEntity {
+	
+	public static final long serialVersionUID = 1;
+	
+	public enum SyncResponseStatus {
+		NEW,
+		PROCESSED
+	}
+	
+	@NotNull
+	@Column(name = "message_uuid", length = 38, nullable = false, unique = true, updatable = false)
+	@Access(AccessType.FIELD)
+	private String messageUuid;
```

Comment:
I think we should not make message_uuid unique because I can imagine a scenario where the receiver might send a response for the same sender message more than once e.g. on a retry attempt in case something went wrong when actually the first attempt went through.

Reply:
Alright. I've made the change.